### PR TITLE
Don't round the revenue stats

### PR DESCRIPTION
**Pull request by nbradbury**
The stats today widget previously rounded the revenue, resulting in showing a different number than the app.

app

This PR corrects the problem.

fix

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/appwidgets/stats/today/TodayWidgetListViewModel.kt [hunk 1]
```diff
@@ -85,6 +84,13 @@ class TodayWidgetListViewModel @Inject constructor(
         )
     }
 
+    private fun formatRevenue(revenue: Double, currencyCode: String) =
+        if (revenue == 0.0) {
+            currencyFormatter.formatCurrencyRounded(revenue, currencyCode)
+        } else {
+            currencyFormatter.formatCurrency(revenue.toBigDecimal(), currencyCode)
+        }
+
```
Comment (atorresveiga) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/appwidgets/stats/today/TodayWidgetListViewModel.kt]: I see that this function has the same logic as [the one on MyStoreStats](https://github.com/woocommerce/woocommerce-android/blob/76dfb3ea47f032ec3ca6aee7d0cd35bd8dc62fea/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/mystore/MyStoreStatsView.kt#L523). I will recommend moving this logic to the CurrencyFormatter to ensure we use the same logic in both cases.

**Review by atorresveiga — APPROVED** [review]
I tested the changes and everything is working as expected. Great work Nick! I leave just a small comment about reusing the same function in MyStore and the Widget to ensure consistency.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/appwidgets/stats/today/TodayWidgetListViewModel.kt (hunk 1 above)
Comment (nbradbury) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/appwidgets/stats/today/TodayWidgetListViewModel.kt]: Good call! I made this change in [0f2b4c4](https://github.com/woocommerce/woocommerce-android/pull/7387/commits/0f2b4c4a6126cb10d65f93d644c3cf391c9325b7)

**Comment by atorresveiga** [comment]
LGTM! :shipit:

**Pull request merged**